app/src/awallet/java/com/alphawallet/app/entity/VisibilityFilter.java

```diff
@@ -98,4 +98,7 @@ public static boolean canChangeWallets()
 
     //In main wallet menu, if wallet allows adding new tokens
     public static boolean canAddTokens() { return true; }
+
+    //Implement minimal dappbrowser with no URL bar.
+    public static boolean minimiseBrowserURLBar() { return false; }
```

Comment:
This is a command function; ```hasVisibleBrowserURLBar``` erroneously indicates this is a variable getter from the method.

The functions here determine how the app appears, rather than fetching info about how they are. Maybe shouldMinimiseBrowserURLBar() would be better?